content/ui-patterns/messaging.mdx

```diff
@@ -7,25 +7,112 @@ import {Box} from '@primer/react'
+Messaging holds differing levels of prominence, which can be used to determine the appropriate component to use. Most messages fall into one of the following categories, from most prominent to least:
 
-**Feedback**: communicates something to the user based on an action they've taken or a change in context
+**System updates**: These types of message originate from GitHub and are not user initiated. A global site wide announcement (outage, missing payment details, critical and blocking) or messaging related to user account/permissions should use the **Announcement** component. System announcements related to a specific product area such as a single sign-on message should use the **Banner** component. System update type messages cannot be dismissed until the issue is resolved, or may be perminent if they are purely informative and not critical.
 
-- _Toasts or flash alerts are best suited for this type of messaging._
+(image here)
 
-**Highlights**: bring attention to how and why to engage with a feature
+**Feedback**: Communicates the result of a user action (submitting a form, toggling a setting, etc.) Feedback can be positive, negative, or a warning. Feedback is shown in either a **Banner** or **Inline message**. For forms, an inline message is used for individual field validation which may be combined with a banner if multiple fields have errors. Banners are placed at the top of the page or section they are related to, but still within the body content. Inline messages are placed near the action they are related to.
 
-- _Popovers are best suited for this type of messaging._
+(image here)
 
-**Updates**: inform the user of necessary updates or changes to the product as it relates to them.
+**Awareness**: Communicates information that is relevant to the user but not necessarily related to an action they've taken. This type of message is typically used to provide context or additional information about a feature or product. A **Callout** might be used to suggest an action or helpful tip, while a **Popover** is typically used to announce a new feature.
 
-- _Flash alerts are best suited for this type of messaging._
+(image here)
+
+## Message states
+
+<img
+  width="960"
+  alt="A group of labels showing the proper icon and color combination for each message state"
+  src="https://github.com/primer/css/assets/18661030/41d3a706-9aa3-4f05-8bad-5176f49a7d5d"
+/>
+
+### Info
+
+Use the `info` state to highlight messaging that may help the user complete a task or provide additional context. This state is typically used for informational purposes and is not critical to the user's experience.
+
+Examples:
+- a setting that impacts the current view
+- nudge towards a new feature
+- a non blocking action to unlock additional features
```

Comment:
```suggestion
- a non-blocking action to unlock additional features
```

@langermank what is this supposed to mean? What is an action that unlocks additional features?

Reply:
Also, should we include visual examples as well? I think this would make it easier to understand. (We can create those once we are set on the examples).

Reply:
Trying to account for inline banners in account settings that are like.. if you do X, you will have access to Y feature. I think I saw some examples of this in our audit 😆 agree this is not very clear and images will help